gtsam/inference/Conditional-inst.h

```diff
@@ -59,10 +59,17 @@ double Conditional<FACTOR, DERIVEDCONDITIONAL>::evaluate(
 
 /* ************************************************************************* */
 template <class FACTOR, class DERIVEDCONDITIONAL>
-double Conditional<FACTOR, DERIVEDCONDITIONAL>::logNormalizationConstant()
+double Conditional<FACTOR, DERIVEDCONDITIONAL>::negLogConstant()
     const {
   throw std::runtime_error(
-      "Conditional::logNormalizationConstant is not implemented");
+      "Conditional::negLogConstant is not implemented");
+}
+
+/* ************************************************************************* */
+template <class FACTOR, class DERIVEDCONDITIONAL>
```

Comment:
Kill this - new philosophy: we are either in probability space, or in neg-log space. Nothing in between.